contracts/common.yml

```diff
@@ -9863,6 +9866,14 @@ components:
       enum:
         - implicit
         - explicit
+    ColorMapping:
+      type: object
+      description: maps series id to the color index that is used to index into the colors array
+      additionalProperties:
+        type: int
+      example:
+        series_id_1: 0
+        series_id_2: 1
```

Comment:
this is a decent example, but it might be more illustrative if the series ids were more like how they'll be in real life using a common example that many people in the company would be familar with:

```
measurement_birdmigration_europe: 0
```

that isn't a very good example, but I'm sure you can find a more common example using sample data or something like that?